docs/website/docs/hub/getting-started/installation.md

```diff
@@ -126,40 +128,45 @@ C:\> .\venv\Scripts\activate
 `dlthub` will be automatically installed with workspace extra:
 
 ```sh
-# install the newest dlt version or upgrade the existing version to the newest one
+# install the newest dlt[workspace] version or upgrade the existing version to the newest one
 uv pip install -U "dlt[workspace]"
 ```
 
-Please install a valid license before proceeding, as described under [licensing](#licensing).
+Please install a valid license before proceeding, as described under [licensing](#self-licensing).
 
-## Licensing
 
-Once you have a valid license, you can make it available to dlthub using one of the following methods:
 
-1. **Environment variable**: set the license key as an environment variable:
+## Self-licensing
```

Comment:
We still need to show people how the license can be used, so maybe you can call the whole section `Licensing`. And inside explain, that use of some of the features requires a license which will be obtained automatically if you purchase a paid dltHub offering. Then stuff about how to put license in a toml/env var (you can use tabs!). And then have a subsection about self-licensing.

Reply:
Also you need to put a link to EULA somewhere